Automattic/woocommerce-payments
# Check for platform checkout eligibility before customer is redirected

#### Changes proposed in this Pull Request

1796-gh-Automattic/woocommerce-payments-server adds a new API endpoint to WCPay Server for getting a merchant's eligibility for platform checkout. This PR adds a function to the API client to send a request to that endpoint. This function will be used by platform checkout to determine if a merchant is eligible.

In addition, if we try to initialize platform checkout and it turns out the merchant _isn't_ eligible, this PR closes the platform checkout iframe popup so that the customer can continue with standard checkout.

#### Testing instructions

This PR should be tested alongside 1796-gh-Automattic/woocommerce-payments-server in WCPay Server and PR 654 for platform checkout. Full testing instructions are in platform checkout's PR 654.

-------------------


**Post merge**
